pkg/hubble/observer/local_observer.go

```diff
@@ -162,6 +162,15 @@ nextEvent:
 		}
 
 		s.GetRingBuffer().Write(ev)
+		for _, f := range s.opts.OnDecodedEvent {
+			stop, err := f.OnDecodedEvent(ctx, ev)
```

Comment:
I think we should move this to before `s.GetRingBuffer().Write(ev)` to be consistent with `OnDecodedFlow`. The stop boolean is a nop here.